### PR TITLE
Changed <br> to get a consistent overlay-pdf form

### LINKED ISSUE #1831: [CSS Bug]: Split pdf by size page improvements
### Installation Method

Docker

### The Problem

image
Spacing should be maintained uniformly at highlighted sections

### Version of Stirling-PDF

0.28.3

### Last Working Version of Stirling-PDF

_No response_

### Page Where the Problem Occurred

http://localhost:8080/overlay-pdf

### Docker Configuration

_No response_

### Relevant Log Output

_No response_

### Additional Information

_No response_

### Browsers Affected

Chrome

### No Duplicate of the Issue

- [X] I have verified that there are no existing issues raised related to my problem.

### PR DESCRIPTION
**Pull request by DrSommer20**
# Description

Edited the HTML for overlay-pdf to get a consistent look. 
Repaired the missing overlay-pdfs.counts.label to be displayed and not deleted on change

Closes #1831 

## Checklist

- [ X] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [ X] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] My changes generate no new warnings
- [ X] I have read the section [Add New Translation Tags](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/HowToAddNewLanguage.md#add-new-translation-tags) (for new translation tags only)

### CODE REVIEW

File: src/main/resources/templates/fragments/common.html [hunk 1]
```diff
@@ -92,7 +92,7 @@
           const downloadCompleteText = /*[[#{downloadComplete}]]*/ 'Download Complete';
           window.downloadCompleteText = downloadCompleteText;
           // Create the 'show-game-btn' button
-          var gameButton = $('<button type="button" class="btn btn-primary" id="show-game-btn" style="display:none;">' + boredWaitingText + '</button><br><br>');
```
Comment (Frooodle) [inline src/main/resources/templates/fragments/common.html]: Does this change look alright on the other pages? have you tested?

**Review by Frooodle — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/fragments/common.html (hunk 1 above)
Comment (DrSommer20) [reply src/main/resources/templates/fragments/common.html]: Yes, there where a "< br >" to much on all forms bevor the Submit Btn.

Example:
Before:
2024-09-08 23_14_17-Stirling PDF - Auto Split PDF

After:
2024-09-08 23_14_25-Stirling PDF - Auto Split PDF


There are still layout issues, but they are not related to the gameButton

Comment (Frooodle) [reply src/main/resources/templates/fragments/common.html]: Perfect, I see there are merge issues
After that and ensuring the change doesn't effect anything we can merge!
Thanks again!

**Review by DrSommer20 — COMMENTED** [review]
Changed to main branch setting for progressbar fix

**Pull request merged**